Dwolla.Client/Models/Requests/CreateBeneficialOwnerRequest.cs

```diff
@@ -0,0 +1,16 @@
+﻿using Newtonsoft.Json;
+using System;
+
+namespace Dwolla.Client.Models.Requests
+{
+    public class CreateBeneficialOwnerRequest
+    {
+        public string FirstName { get; set; }
+        public string LastName { get; set; }
+        [JsonConverter(typeof(DateOnlyConverter))]
```

Comment:
Is this required? This is the same format `CreateCustomerRequest` uses and that doesn't need a `JsonConverter`.

Reply:
I don't know why it's required, but I receive a "Invalid format" error from the api if it is not present. The JSON includes a time string by default.

Reply:
Ok, just talked to the developers that wrote this endpoint. They're more strict on the format and it's a breaking change to modify Customers so we'll continue to allow the time there. So this is needed, thanks for adding it!